packages/ripple-binary-codec/HISTORY.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Added
+* Updated to include latest updates to `definitions.json` including changes for the `fixNFTokenRemint` amendment.
```

Comment:
nit: note that this includes AMM definitions updates too.

Reply:
Shouldn't the AMM definitions wait for the AMM PR instead of being done here?

Reply:
I'd prefer to just match rippled's `develop` as best as possible instead of manually editing the `definitions.json` file. Plus, I don't think there's any harm in having more `definitions` available.

Reply:
Can we hold off on this PR until that's merged in, then? Since that one's going to be merged soon anyways.

Reply:
This was generated from `develop` already - not sure why the definitions were there before the big AMM PR, but it seems they're already defined.